app/views/static_pages/lessons.html.erb

```diff
@@ -1,5 +1,32 @@
 <% provide(:title, "Lessons") %>
+<p></p>
```

Comment:
@shaniajubelag Same comment in https://github.com/framgia/ph_sels_rails_shania/pull/8/files#diff-39ce76d94a7eb2d6282c0ff8a4ce44a9R2